dataactbroker/handlers/accountHandler.py

```diff
@@ -408,7 +408,7 @@ def listSubmissionsByCurrentUserAgency(self):
             error_count = self.interfaces.errorDb.sumNumberOfErrorsForJobList(jobIds)
             submissionDetails.append({"submission_id": submission.submission_id, "last_modified": submission.updated_at.strftime('%m/%d/%Y'),
                                       "size": total_size, "status": status, "error": error_count})
-        return JsonResponse.create(StatusCode.OK, {"recent": submissionDetails})
+        return JsonResponse.create(StatusCode.OK, {"submissions": submissionDetails})
```

Comment:
Since the doc updates are coming in a future PR, just checking to make sure the front-end team is cool with this response (I agree that _submissions_ is much better than _recent_).